src/types.ts

```diff
@@ -107,3 +117,14 @@ export interface CodeTracker {
   newline(): void;
   literal(literal: string): void;
 }
+
+// TODO(jared): Remove hack by contributing to the types upstream.
+export type System = VF.System & {
+  getStaves(): VF.Stave[];
+  getX(): number;
+  setX(x: number): void;
+  getY(): number;
+  setY(y: number): void;
+  addVoices(voices: VF.Voice[]): void;
+  appendSystem(width: number): System;
+};
```

Comment:
Why is that necessary?

Reply:
`VF.System` needs to be updated to include the additional types. Without this, this causing type errors.

Reply:
All you need to do is to install the vexflow pre release under ./vexflow/

Reply:
`yarn add ./vexflow/vexflow-5fa72362.tgz`

Reply:
This might also resolve the other two comments